recipes/rcat/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "RCAT" %}
+{% set version = "v1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jplind79/{{ name }}/archive/{{ version }}.tar.gz
```

To keep things working when `v` is removed from `version`
```suggestion
  url: https://github.com/jplind79/{{ name }}/archive/v{{ version }}.tar.gz
```